compiler-core/src/format.rs

```diff
@@ -834,7 +834,18 @@ impl<'comments> Formatter<'comments> {
                     .expect("Function in a statement must be named")
                     .1,
             )
-            .append(self.wrap_arguments(arguments, function.location.end));
+            .append(
+                self.wrap_arguments(
+                    // We count end location of arguments here.
+                    // Before this change, wrap_arguments consumed all comments
+                    // leading in moving comments from return annotation to argument list
```

Comment:
Document the current code, not the code of the past!

What does "we count end location of arguments here" mean?